samples/net/vlan/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _vlan-sample:
+
+Virtual LAN Sample Application
+##############################
+
+Overview
+********
+
+The VLAN sample application for Zephyr will setup two virtual LAN networks.
+The application sample enables net-shell and allows users to view VLAN settings.
+
+The source code for this sample application can be found at:
+:file:`samples/net/vlan`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
+Building and Running
+********************
+
+A good way to run this sample is to run this VLAN application inside QEMU
+as described in :ref:`networking_with_qemu`.
+Currently only one VLAN network (tag) is supported when Zephyr is run inside
+QEMU. If using FRDM-K64F board, then multiple VLAN networks can be configured.
+Note that VLAN is only supported for boards that have ethernet port.
+
+Follow these steps to build the VLAN sample application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/vlan
+   :board: <board to use>
+   :conf: prj.conf
+   :goals: build
+   :compact:
+
+The default configuration file prj.conf creates two virtual LAN networks
+with these settings:
+- VLAN tag 100: IPv4 198.51.100.1 and IPv6 2001:db8:100::1
```

Comment:
add a blank line before a list (otherwise the line looks like a continuation of the sentence)